ext/pgsql/tests/80_bug24499.phpt

```diff
@@ -48,6 +46,13 @@ pg_close($dbh);
 
 echo "Done\n";
 
+?>
+--CLEAN--
+<?php
+require_once('inc/config.inc');
+$dbh = pg_connect($conn_str);
+
+pg_query($dbh, "DROP TABLE id CASCADE");
```

Comment:
Probably should be `DROP TABLE IF EXISTS`

Reply:
Fixed in https://github.com/php/php-src/pull/12608/commits/8a3484888dce8530ada1c4c698a1923f180020ca
All existing code unrelated to this pull request was also fixed.